src/main/java/com/panda/annotations/LoaderAnnotation.java

```diff
@@ -1,9 +1,13 @@
 package com.panda.annotations;
 
+import com.panda.utility.Loader;
+
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
 public @interface LoaderAnnotation {
-    public String path() default "";
+    @Deprecated
+    String path() default "";
+    Class<? extends Loader> loader() default Loader.class;
 }
```

Comment:
Maybe have a field that can be used to define if `isLoader`.

Reply:
This is only because in `AnnotationLoader` logic to check if `loader()` returns `Loader.class` is used, and can easily be replaced by something like: `if (annotation.isLoader())`